README.md

```diff
@@ -594,6 +543,28 @@ the correct function to yank in vterm buffers.
 (advice-add 'counsel-yank-pop-action :around #'vterm-counsel-yank-pop-action)
 ```
 
+### How can I get the local directory without shell-side configuration?
+
+We recommend that you set up shell-side configuration for reliable directory
+tracking. If you cannot do it, a possible workaround is the following.
+
+On most GNU/Linux systems, you can read current directory from `/proc`:
+```emacs-lisp
+(defun vterm-directory-sync ()
+  "Synchronize current working directory."
+  (interactive)
+  (when vterm--process
+    (let* ((pid (process-id vterm--process))
+           (dir (file-truename (format "/proc/%d/cwd" pid))))
```

Comment:
I found `(dir (file-truename (format "/proc/%d/cwd/" pid))))` is better

Reply:
That's what we have here. Am I missing something?

Reply:
`/proc/%d/cwd/` with an additional `/` makes it prettier in `counsel`.

![no trialing slash](https://user-images.githubusercontent.com/4024656/83965204-73617c80-a8e4-11ea-95f0-98a66b129cd9.png)

![prettier](https://user-images.githubusercontent.com/4024656/83965219-94c26880-a8e4-11ea-8e25-b1729eb27b57.png)

Reply:
Thanks for the clarification